actions/createGame.tsx

```diff
@@ -0,0 +1,19 @@
+const createGame = (event) => {
```

Comment:
That file should be `.ts` file, not a `.tsx` as it does not return any Type-strict JSX element/s.

Also, event is not type-safe, you should be able to type it